db/schema.rb

```diff
@@ -19,8 +19,17 @@
     t.string "title"
   end
 
+  create_table "reviews", force: :cascade do |t|
+    t.string "user"
+    t.text "body"
+    t.integer "rating"
+    t.bigint "book_id"
```

Comment:
`user` isn't using the users table that we have here. Should change this so that it includes a `user_id` and acts as a join.